src/Router.tsx

```
@@ -1,23 +1,23 @@
-import { Friend } from "./pages/Friend";
-import { Room } from "./pages/Room";
-import { ConcertRoom } from "./pages/ConcertRoom";
-
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
+import { Header } from './components/Header'
+import { MyPage } from './pages/mypage/Mypage'
+import { EditMyPage } from './pages/mypage/Edit'
+import { Friend } from './pages/Friend'
+import { Room } from './pages/Room'
+import { ConcertRoom } from './pages/ConcertRoom'
+import { Drum } from './pages/Drum'
```

ESLint

[error] 'Drum' is defined but never used.

(@typescript-eslint/no-unused-vars)